mietechnologies/Columbia-City-Christian-Church-iOS
# Added ProfileScreen

## Description



### Type of Change


## Does this introduce a breaking change?



## How Can This PR Be Tested?



1.) Build and Run the application.
2.) Tap on the top right icon, which looks like a profile image.
3.) Confirm that the view looks as expected.
4.) Tap the Edit button on the Profile Image, select and test cropping a photo. Confirm that it works as expected.
5.) Once satisfied with the testing of this screen, tap the back button (or swipe back), and confirm the toolbar buttons look the same and do not have their tops clipped.

![Simulator Screenshot - iPhone 15 Pro - 2024-06-25 at 22 34 33](https://github.com/mietechnologies/Columbia-City-Christian-Church-iOS/assets/22135180/7436ec93-81d4-4b30-8b46-85da3a2b947e)

**Test Configuration**:
* Firmware version: 17.5
* Hardware: iPhone 15 Pro
* Is Simulator:

## Checklist:


- [x] Doc strings have been added to any public APIs
- [x] The CHANGELOG.md file has been updated with your changes (1 entry per PR)
- [x] Before merging, rebase your branch with the target branch
